language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -344,23 +444,27 @@ genDefDataType curPkgId conName mod tpls def =
         -- definition.
         serFromDef f c2 = f . snd . fst . genDefDataType curPkgId (conName <.> c2) mod tpls
 
+infixr 6 <.> -- This is the same fixity as '<>'.
+(<.>) :: T.Text -> T.Text -> T.Text
+(<.>) u v = u <> "." <> v
+
 genType :: ModuleName -> Type -> (T.Text, T.Text)
 genType curModName = go
   where
     go = \case
-        TVar v -> dup (unTypeVarName v)
+        TVar v -> dupe (unTypeVarName v)
```

Comment:
Renamings like this in such a big PR make the PR even harder to review. 😓

Reply:
Ok. I'm sorry.